src/Compilers/VisualBasic/Test/Emit/BasicCompilerEmitTest.vbproj

```diff
@@ -216,24 +112,37 @@
     <Content Include="ExpressionTrees\Results\XmlLiteralsInExprLambda01_Result.txt" />
     <Content Include="ExpressionTrees\Results\XmlLiteralsInExprLambda02_Result.txt" />
     <Content Include="ExpressionTrees\Results\XmlLiteralsInExprLambda03_Result.txt" />
+  </ItemGroup>
+  <ItemGroup>
+    <Content Include="CodeGen\ConversionsILGenTestSource.vb" />
```

Comment:
These are the content items that are vb files and hence have corresponding Compile removes. So I moved them to the same itemgroup to keep them together. Open to other organization suggestions.